schemas/subs/vc-common.js

```diff
@@ -0,0 +1,121 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {config} = require('bedrock');
+const {schemas} = require('bedrock-validation');
+
+const {constants} = config;
+
+const proof = () => ({
+  type: 'object',
+  title: 'JSON LD Proof',
+  required: [
+    'type',
+    'created',
+    'proofPurpose',
+    'verificationMethod',
+    'jws'
+  ],
+  properties: {
+    type: {type: 'string'},
+    proofPurpose: {
+      type: 'string',
+      examples: [
+        'authentication',
+        'capabilityDelegation'
+      ]
+    },
+    verificationMethod: {type: 'string'},
+    created: schemas.w3cDateTime(),
+    challenge: {type: 'string'},
+    jws: {type: 'string'}
+  }
+});
+
+// add the capabilitychain for the CapabilityDelegationProof.
+const capabilityDelegationProof = () => {
+  const _proof = proof();
```

Comment:
What do you think about `Object.assign({}, proof(), {...});`

I think that would be more legible and maintainable.

Reply:
1. there is no need to create a new object as the proof function creates a new object each time.
2. the object syntax makes doing things like pushing to an array harder to do.

so I think I will pass on this suggestion unless there is something I'm not seeing. Is that ok?